packages/frontend/src/components/common/Filters/configs.ts

```diff
@@ -148,26 +150,36 @@ export const getFilterRuleLabel = (
                 filterRule.operator === FilterOperator.IN_THE_PAST ||
                 filterRule.operator === FilterOperator.IN_THE_NEXT
             ) {
-                valuesText = `${filterRule.values?.[0]} ${
-                    filterRule.settings.completed ? 'completed ' : ''
-                }${filterRule.settings.unitOfTime}`;
+                if (isFilterRule(filterRule)) {
+                    valuesText = `${filterRule.values?.[0]} ${
+                        filterRule.settings.completed ? 'completed ' : ''
+                    }${filterRule.settings.unitOfTime}`;
+                } else {
+                    valuesText = `${filterRule.values?.[0]}`;
+                }
             } else if (filterRule.operator === FilterOperator.IN_BETWEEN) {
                 valuesText = `${filterRule.values?.[0]} and ${filterRule.values?.[1]}`;
             } else if (filterRule.operator === FilterOperator.IN_THE_CURRENT) {
-                valuesText = `${filterRule.settings.unitOfTime.substring(
-                    0,
-                    filterRule.settings.unitOfTime.length - 1,
-                )}`;
+                if (isFilterRule(filterRule)) {
+                    valuesText = `${filterRule.settings.unitOfTime.substring(
+                        0,
+                        filterRule.settings.unitOfTime.length - 1,
+                    )}`;
+                } else {
+                    valuesText = `${filterRule.values?.[0]}`;
+                }
```

Comment:
I am reusing filter methods because conditional formatting uses components from the filters. and they are basically the same. I created a "supertype" called ConditionalRule which is a supertype for `ConditionalFormattingRule` and `FilterRule` (they both extend from it). I am not sure if it is a good idea to reuse this `getFilterRuleLabel` (now named as `getConditionalRuleLabel`) but again, it is very same. except the date difference which conditional formatting does not support (yet?)